src/index.ts

```diff
@@ -10,18 +11,25 @@ const dotsimus = new Client({
   ],
 });
 
-if (!process.env.DISCORD_TOKEN) {
-  throw new Error("Discord token not provided.");
+
+if (!process.env.NODE_ENV) {
+  process.env.NODE_ENV = 'production';
+}
+
+if (!process.env.BOT_TOKEN) {
+  throw new Error(
+    "Bot token not supplied. Please confirm it is set in the .env file!"
+  );
 }
 
-dotsimus.login(process.env.DISCORD_TOKEN);
+dotsimus.login(process.env.BOT_TOKEN);
 
-dotsimus.on("ready", (client) => {
-  dotsimus.emit("debug", `Connected to Discord as ${client.user.tag}`);
+dotsimus.on('ready', (client) => {
+  dotsimus.emit('debug', `Connected to Discord as ${client.user.tag}`);
 });
 
-if (process.env.NODE_ENV === "development") {
-  dotsimus.on("debug", (message) => {
+if (process.env.NODE_ENV == 'development') {
```

Comment:
Either change it in the prettier config to use single quotes, or continue using double quotes please